recipes/lfzip-nlms/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cd src/
+$CXX nlms_helper.cpp -std=c++11 -Wall -O3 $CFLAGS -o nlms_helper.out $LDFLAGS
+cp nlms_helper.out $PREFIX/bin
+(echo "#!/usr/bin/env python3" && cat nlms_compress.py) > nlms_compress_1.py
```

Comment:
Can this be done upstream?

Reply:
Sorry, I don't understand what upstream refers to here

Reply:
I mean at https://github.com/shubhamchandak94/LFZip